src/main/java/com/opencart/pages/MyAccountPage.java

```diff
@@ -0,0 +1,32 @@
+package com.opencart.pages;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class MyAccountPage extends HeaderPage {
+    @FindBy(xpath = ".//*[contains(@href, 'edit')]")
+    private WebElement editAccountButton;
+
+    public WebElement getEditAccountButton() throws InterruptedException {
+        Thread.sleep(5000);
+        return editAccountButton;
+    }
+
+    @FindBy(xpath = ".//div[@class='alert alert-success alert-dismissible']")
```

Comment:
you can use xpath - contains() function and define just a part of the full class name in your locator